topics/variant-analysis/tutorials/baculovirus-isolate-variation/tutorial.md

```diff
@@ -0,0 +1,851 @@
+> (family *Baculoviridae*, genus *Alphabaculovirus*, species *Alphabaculovirus aucalifornicae*) is one of the 
+> best-studied baculovirus genomes and is 133,894 bp long ({% cite Ayres1994 %}). 
+> It is the first fully sequenced genon of baculoviruses and today hundreds of genomes are fully sequenced
+> and publically available at NCBI Genbank. The genome of AcMNPV-C6 encodes for 154 open reading frames (ORF)
+> in both forward and reverse orientation (see figure). 
+> So far, 38 ORFs were identified, which are encoded by all baculoviruses sequenced to date and are 
+> called baculovirus core genes. In addition to the core genes, genes were found 
+> that are encoded by baculoviruses from the genera *Alphabaculovirus* and *Betabaculovirus*. 
+> 
+> ![Alternative text](https://ictv.global/sites/default/files/inline-images/7651.ODD.Baculo.Fig2.v2.png "Figure: Schematic representation of the AcMNPV-C6 genome. ORFs and their orientation are indicated by arrows. (Figure from {% cite ICTV_AcMNPV_genome_figure %})")
+>
+{: .comment}
+
+The genome size makes it difficult to study genetic variation within a baculovirus population, 
+especially since the most commonly used sequencing technique (Illumina sequencing) 
+generates only short reads and requires genome assembly. Since genome assembly generates 
+a consensus sequence that reflects the majority of a sequenced virus population, 
+occuring genetic variation is masked. Tools for haplotype-sensitive assembly are 
+available but so far have been establisehd for viruses with a relatively short genome 
+but not for baculoviruses or other large dsDNA viruses (REFERENCES). Using Nanopore sequencing, 
+it is possible to sequence signficiant fragments of  baculovirus genomes to determine major 
+haplotypes ({% cite Wennmann2024 %}). 
+
+Single nucleotide variants (SNV) have been proven as a powerful tool for analyzing 
+the genetic variability of sequenced baculovirus isolates ({% cite Fan2020 %}). SNV 
+are particularly useful for the analysis of intra-isolate or intra-sample specific 
+variation, as a many bioinformatic workflows and tools are established for SNV determination 
+and processing. For the identification of SNVs, Illumina sequencing data is mostly used, 
+as this sequencing technique provides highly accurate reads with a very low probability 
+of sequencing error. The data is usually provided and stored in fatsq or fastqsanger format.
+
+When a baculovirus sample or isolate is sequenced, a dataset of sequence reads is 
+obtained that consists of a multitude of fragmented genomes and the excact reconstruction 
+of individual whole genomes is no longer possible. However, variable SNV positions can 
+be used as markers to count the frequency of the nucleotides that occur in the SNV positions. 
+The determination of the nucleotides is only possible because not one but many genomes 
+were sequenced. Therefor, the SNVs reflect the genetic variability of the virus population itself. 
```

Comment:
```suggestion
When a baculovirus sample or isolate is sequenced, a dataset of sequence reads is 
obtained that consists of a multitude of fragmented genomes and the exact reconstruction 
of individual whole genomes is no longer possible. However, variable SNV positions can 
be used as markers to count the frequency of the nucleotides that occur in the SNV positions. 
The determination of the nucleotides is only possible because not one but many genomes 
were sequenced. Therefore, the SNVs reflect the genetic variability of the virus population itself. 
```